idb/postgres/postgres_integration_test.go

```diff
@@ -545,23 +457,14 @@ func TestIgnoreDefaultFrozenConfigUpdate(t *testing.T) {
 	///////////
 	// Given // A new asset with default-frozen = true, and AccountB opting into it.
 	///////////
-	createAssetNotFrozen := test.MakeAssetConfigTxn(
-		0, total, uint64(6), false, "icicles", "frozen coin", "http://antarctica.com",
-		test.AccountA)
-	modifyAssetToFrozen := test.MakeAssetConfigTxn(
-		assetid, total, uint64(6), true, "icicles", "frozen coin", "http://antarctica.com",
-		test.AccountA)
-	optin := test.MakeAssetOptInTxn(assetid, test.AccountB)
-
-	block, err := test.MakeBlockForTxns(
-		test.MakeGenesisBlock().BlockHeader, &createAssetNotFrozen, &modifyAssetToFrozen,
-		&optin)
+	vb, err := test.ReadValidatedBlockFromFile("test_resources/validated_blocks/IgnoreDefaultFrozenConfigUpdate.vb")
 	require.NoError(t, err)
+	block := ledgercore.MakeValidatedBlock(vb.Blk, vb.Delta)
 
 	//////////
 	// When // We commit the round accounting to the database.
 	//////////
-	err = proc(&rpcs.EncodedBlockCert{Block: block})
+	err = db.AddBlock(&block)
```

Comment:
nit: consider moving these lines next to each other since they'll need to be updated together during the interface migration.